docs/src/installing.rst

```diff
@@ -96,7 +96,7 @@ local copy of Iris::
   conda activate iris-dev
 
 The ``--force`` option is used when creating the environment, this is optional
-and will force the any existing ``iris-dev`` conda environment to be deleted
+and will force that any existing ``iris-dev`` conda environment to be deleted
```

Comment:
```suggestion
and will force any existing ``iris-dev`` conda environment to be deleted
```
It's good to change this, but I think the changed version reads wrong. 
I'd propose to just remove the exatr word entirely.
Do you think that works?